Makefile

```diff
@@ -49,7 +49,7 @@ setup_env:
 	if [[ $$(uname) != "Linux" ]] && [[ $$(uname) != "Darwin" ]]; then \
 		poetry install --only dev; \
```

Comment:
why do we have this actually ? is it for windows ? but why the `--only` ?

Reply:
I have no idea

Reply:
I checked with the autor and it's for windows support.
Not sure how tested this is tbh